examples/priorAproval-largerExtension.json

```diff
@@ -0,0 +1,1613 @@
+{
+  "data": {
+    "user": {
+      "role": "applicant"
+    },
+    "applicant": {
+      "type": "individual",
+      "name": {
+        "first": "William",
+        "last": "Zane"
+      },
+      "email": "areyouon@email.org",
+      "phone": {
+        "primary": "01234000000"
+      },
+      "address": {
+        "sameAsSiteAddress": true
+      },
+      "siteContact": {
+        "role": "applicant"
+      }
+    },
+    "property": {
+      "address": {
+        "latitude": 51.4566948,
+        "longitude": -0.0731039,
+        "x": 533971,
+        "y": 174840,
+        "title": "23, FELLBRIGG ROAD, LONDON",
+        "source": "Ordnance Survey",
+        "uprn": "200003445289",
+        "usrn": "22500947",
+        "pao": "23",
+        "street": "FELLBRIGG ROAD",
+        "town": "LONDON",
+        "postcode": "SE22 9HQ",
+        "singleLine": "23, FELLBRIGG ROAD, LONDON, SOUTHWARK, SE22 9HQ"
+      },
```

Comment:
nit: looks like this example address breaks our "blue plaque" pattern - are we cool with that? Was that too much of an extra barrier/ask for folks generating examples?